pkg/machines/libvirt-dbus.js

```diff
@@ -1043,10 +1045,19 @@ function startEventMonitorDomains(connectionName, dispatch) {
                 dispatch(getVm({
                     connectionName,
                     id: objPath,
-                    updateOnly: true,
-                }));
-                // transient VMs don't have a separate Undefined event, so remove them on stop
-                dispatch(removeVm({ connectionName, id: objPath, transientOnly: true }));
+                }))
+                        .catch(() => {
```

Comment:
Shouldn't this test the actual error? It could have a different reason than a transient resource? Same below.

Reply:
Comment makes sense, I will adjust it accordingly.